platformerLib/GreySquareActor.cpp

```diff
@@ -63,8 +63,10 @@ void GreySquareActor::ApplyMoveSpeed(const oxygine::UpdateState& us)
 void GreySquareActor::Jump()
 {
    auto speed = GetSpeed();
-   if(speed.y > 0 * kPixelPerSecond) speed.y *= 0;
-   SetSpeed(speed + kJumpSpeed);
+   if (speed.y.Value() == 0) {
```

Comment:
Unit::operator== uses epsilon to safely compare floating point numbers, utilize it e.g. by:
speed.y == 0 * kPixelPerSecond

Additionally it makes sens to add kZeroSpeed/kZeroPixelPerSecond, kZeroDistance/kZeroPixel, ... constants, what do you think?